src/query/settings/src/lib.rs

```diff
@@ -157,7 +157,8 @@ impl Settings {
             },
             // max_memory_usage
             SettingValue {
-                default_value: UserSettingValue::UInt64(memory_info.total * 80 / 100),
+                // unit of memory_info.total is kB
+                default_value: UserSettingValue::UInt64(1024 * memory_info.total * 80 / 100),
```

Comment:
Not sure this display is always KB not depends on OS?

Reply:
for other platforms, the value will be convert to Kb

Reply:
e.g.

windows:
https://github.com/FillZpp/sys-info-rs/blob/master/c/windows.c#L106-L113

darwin:

https://github.com/FillZpp/sys-info-rs/blob/master/c/darwin.c#L108-L134